git-cliff/src/lib.rs

```diff
@@ -77,6 +81,56 @@ fn check_new_version() {
 	}
 }
 
+/// Process submodules and add commits to release.
+fn process_submodules(
+	repository: &'static Repository,
+	release: &mut Release,
+) -> Result<()> {
+	// Retrieve first and last commit of a release to create a commit range.
+	let first_commit = release
+		.previous
+		.as_ref()
+		.and_then(|previous_release| previous_release.commit_id.clone())
+		.and_then(|commit_id| repository.find_commit(&commit_id));
+	let last_commit = release
+		.commit_id
+		.clone()
+		.and_then(|commit_id| repository.find_commit(&commit_id));
+	let commit_range = first_commit.zip(last_commit);
```

Comment:
This commit range generation was initially wrong, I needed to use the tag commit id from the previous and current release.